game/beatmap/loader/cbf.lua

```diff
@@ -675,4 +675,19 @@ function cbfLoader:getBackground()
 	return nil
 end
 
+-- Unnecessary, but since it a folder, We may have to look a bit...
+function cbfLoader:getLyrics()
+	local internal = Luaoop.class.data(self)
+	local lyrics = nil
+
+	if Util.fileExists(internal.path.."lyrics.srt") then
+		lyrics = love.filesystem.newFileData(internal.path.."lyrics.srt")
+	elseif Util.fileExists(internal.path.."lyrics.srt.gz") then
+		local temp = love.filesystem.newFileData(internal.path.."lyrics.srt")

```

Comment:
Shouldn't this be `lyrics.srt.gz`?

Reply:
Changed on [e30acfb](https://github.com/DarkEnergyProcessor/livesim2_async/pull/33/commits/e30acfb9a403dbbf4b765d986863d91af223dc94)